com.hoptodesk.HopToDesk.metainfo.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.hoptodesk.HopToDesk</id>
+  
+  <name>HopToDesk</name>
+  <summary>Free Remote Desktop Software. Easily access remote devices securely with end-to-end encryption.</summary>
+
+  <developer id="com.hoptodesk">
+    <name>Begonia Holdings</name>
+  </developer>
+  
+  <developer_name>Begonia Holdings</developer_name>
+  
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>AGPL-3.0-only</project_license>
+  
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+  
+  <description>
+    <p>
+      HopToDesk is a free remote desktop tool allowing users to share their screen, transfer files, and allow remote control access to their computers and devices using end-to-end encryption.&quot;
+    </p>
+  </description>
+
+  <url type="homepage">https://www.hoptodesk.com</url>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/bakabtw/hoptodesk-flatpak/main/screenshot.png</image>
+      <caption>A caption</caption>
+    </screenshot>
+  </screenshots>
+  
+  <launchable type="desktop-id">com.hoptodesk.HopToDesk.desktop</launchable>
+
+  <content_rating type="oars-1.1" />
+
+  <releases>
+    <release version="1.41.8" date="2024-04-20">
+      <url type="details">https://gitlab.com/hoptodesk/hoptodesk/-/commit/d8839edffaebb27b38a3c6a4f5bf353b33d84d9b</url>
```

Comment:
this is not an informative link. This should be a link to the release notes. Or left out. you can also add release notes in `<description>`

Reply:
Removed unused tags